content/docs/getting-started.md

```diff
@@ -28,24 +28,150 @@ wget https://github.com/containerd/containerd/archive/v{{< latest >}}.zip
 unzip v{{< latest >}}.zip
 ```
 
-This will extract the containerd source into a `containerd-{{< latest >}}` directory. If you are using systemd, there is a `containerd.service` file at the root of that directory that you can use.
+This will extract the containerd source into a `containerd-{{< latest >}}` directory. Afterwards, you can run the following commands to start the containerd service using `containerd.service` file present at the root of extracted directory and also enable the containerd servoce so that systemd can automatically start the service at boot time.
+
+```shell
+sudo systemctl start containerd.service
+sudo systemctl enable containerd.service
```

Comment:
These two commands can be merged into `sudo systemctl enable --now containerd.service`

Reply:
@AkihiroSuda , I merged systemctl commands as you suggested.